opel-agent/values.yaml

```diff
@@ -16,17 +16,19 @@ opentelemetry-collector:
       fieldRef:
         apiVersion: v1
         fieldPath: spec.nodeName
-  #- name: CORALOGIX_ENDPOINT
+  #- name: CORALOGIX_TRACES_ENDPOINT
   #  value: <The Coralogix traces ingress endpoint, must be configured>
+  #- name: CORALOGIX_METRICS_ENDPOINT
+  #  value: <The Coralogix metrics ingress endpoint, must be configured>
     
   config:
     exporters:
       coralogix:
-        endpoint: "${CORALOGIX_ENDPOINT}"
+        endpoint: "${CORALOGIX_TRACES_ENDPOINT}"
         private_key: "${CORALOGIX_PRIVATE_KEY}"
         application_name: "${APP_NAME}"
-    receivers:
-      prometheus: null # If you want to enable the prometheusrules/servicemonitor, this line must be deleted, and also the line above ['receivers:']
+        metrics:
+          endpoint: "${CORALOGIX_METRICS_ENDPOINT}"
```

Comment:
Why not just user giving the suffix? and we will control on the prefix wether is logs / metrics /traces

Reply:
I think it might be confusing, they might confuse and put the whole endpoint instead of just part of it, or miss some of it, I think it should be all or nothing, maybe only the region itself like i did in firehose, but i think putting `eu2.coralogix.com:443` can be a bit confusing.

Reply:
Okay, can you try without the port?

Reply:
here I commented only about the endpoint, not related to the port. 
about the port, as I commented above: if I delete it, it throws missing port in address